Source/Harmony/AttachRecepies.cs

```diff
@@ -19,6 +19,8 @@ public static void Postfix()
             foreach (var humanoidRace in humanoidRaces)
             {
                 Log.Message("Adding in draw blood");
```

Comment:
That's a Log.Message for every race added; about fifty lines of logspam on a heavily modded game.

```suggestion
                
```